react/src/components/awards/SprkAward.js

```diff
@@ -78,38 +82,57 @@ const SprkAward = (props) => {
         })}
       </SprkStack>
 
-      <SprkToggle
-        title={disclaimerTitle}
-        analyticsString={disclaimerAnalytics}
-        additionalClasses="sprk-o-Stack__item"
-      >
-        <p className="sprk-b-TypeBodyFour sprk-u-pts sprk-u-pbs">
-          {disclaimerText}
-        </p>
-      </SprkToggle>
+      {(disclaimerTitle || disclaimerText) && (
```

Comment:
> Should update React component to not render the toggle if no toggle info is supplied.

If I had disclaimerTitle but no disclaimer text, then wouldn't an empty Toggle render?

I would expect this to need both Title and Text by using &&

Reply:
Good question. I'll get some clarity on what the AC is going for.

Reply:
confirmed with @afebbraro - we're going to only show the Toggle if there is valid disclaimerText AND disclaimerTitle for both Angular and React.